docs/reference/workflows/ecs/concepts/entity-contracts.md

```diff
@@ -0,0 +1,51 @@
+<%(TOC)%>
+
+# ECS: Entity contracts
+
+<%(Callout message="
+Before reading this document, make sure you are familiar with:
+
+* [Temporary components]({{urlRoot}}/reference/workflows/ecs/concepts/temporary-components)
+* [Worker entity]({{urlRoot}}/reference/workflows/ecs/worker-entity)
+* [System update order]({{urlRoot}}/reference/workflows/ecs/concepts/system-update-order)
+")%>
+
+This documentation describes the guarantees we provide for the components that an ECS Entity has.
+
+## Worker entities
+
+A worker entity is created for each ECS World that is associated with a [worker]({{urlRoot}}/reference/concepts/worker).
+
+The following guarantees hold for worker entities:
+
+* The worker entity will exist for as long as the ECS World exists.
+* A [`WorkerEntityTag`]({{urlRoot}}/api/core/worker-entity-tag) component is always available on the worker entity.
+* An [`OnConnected`]({{urlRoot}}/api/core/on-connected) temporary component is added upon creating the worker entity.
+* An [`OnDisconnected`]({{urlRoot}}/api/core/on-disconnected) temporary component is added after the connection to the SpatialOS Runtime has been lost.
```

Comment:
these are also reactive components, are we still using them even when they are disabled?

Reply:
yes they're being used - hence there's a distinction between "temporary" components and "reactive" components

Reply:
Temporary != reactive.